mailbox/store/src/test/java/org/apache/james/mailbox/store/PartContentBuilderMultipartAlternativeTest.java

```diff
@@ -73,27 +73,27 @@
 
     PartContentBuilder builder;
 
-    @Before
-    public void setUp() throws Exception {
+    @BeforeEach
+    void setUp() {
         builder = new PartContentBuilder();
     }
 
     @Test
-    public void testShouldLocatePartsOfMultipartAlterative() throws Exception {
+    void testShouldLocatePartsOfMultipartAlterative() throws Exception {
         assertThat(bodyContent(MULTIPART_ALTERNATIVE, 1)).isEqualTo(ALT_PLAIN_BODY);
         assertThat(bodyContent(MULTIPART_ALTERNATIVE, 2)).isEqualTo(ALT_HTML_BODY);
         assertThat(bodyContent(MULTIPART_ALTERNATIVE, 3)).isEqualTo(ALT_XHTML_BODY);
     }
 
     @Test
-    public void testShouldLocateHeadersOfMultipartAlterative() throws Exception {
+    void testShouldLocateHeadersOfMultipartAlterative() throws Exception {
         checkContentType(CONTENT_TYPE_PLAIN, MULTIPART_ALTERNATIVE, 1);
         checkContentType(CONTENT_TYPE_HTML, MULTIPART_ALTERNATIVE, 2);
         checkContentType(CONTENT_TYPE_XHTML, MULTIPART_ALTERNATIVE, 3);
     }
-
+private
```

Comment:
typo?

Reply:
Good catch ! somehow it doesn't fail the compilation neither running the tests, but then the last test doesn't run because of this. I'm surprised the build can run with that though...

Reply:
I squashed it directly, as it's minor